skycoin/skywire
# Windows build

Did you run `make format && make check`? yes

Fixes #778 

 Changes:	
- Various fixes to make skywire build on windows

TODO:

- [x] `cmdutil` function such as `cmdutil.SignalContext` should probably be fixed on `dmsg` repo
- [x] many things are redundant (see `stdio` package for example) in this draft PR, probably refactor it later once it is tested

How to test this PR:
- Follow https://github.com/skycoin/dmsg/pull/83#issue-655883397
- See skywire wiki for running it on windows

NEEDS
- https://github.com/skycoin/dmsg/pull/93
